src/codec/id.rs

```diff
@@ -907,7 +923,7 @@ impl From<AVCodecID> for Id {
 			AV_CODEC_ID_HDMV_TEXT_SUBTITLE => Id::HDMV_TEXT_SUBTITLE,
 
 			/* other specific kind of codecs (generally used for attachments) */
-			AV_CODEC_ID_TTF       => Id::TTF,
+			AVCodecID_AV_CODEC_ID_TTF       => Id::TTF,
```

Comment:
Thanks for looking into this :)

You can ignore those, I am already ignoring AVRational (https://github.com/meh/rust-ffmpeg-sys/pull/41/files#diff-a7b0a2dee0126cddf994326e705a91eaR637, I'm ignoring and leaving manually written code as generated bindings do not derive Eq and PartialEq for AVRational). Same could be done with those constants.